shell/platform/android/io/flutter/embedding/android/FlutterFragmentActivity.java

```diff
@@ -293,7 +293,8 @@ private void switchLaunchThemeForNormalTheme() {
     } catch (PackageManager.NameNotFoundException exception) {
       Log.e(
           TAG,
-          "Could not read meta-data for FlutterFragmentActivity. Using the launch theme as normal theme.");
+          "Could not read meta-data for FlutterFragmentActivity. Using the launch theme as normal"
```

Comment:
You can revert this. Totally unnecessary; the internal tooling for some reason made this change. It should be reverted in google3 as well (which will happen automatically with the roll).

Reply:
Done.